doc/classes/NavigationObstacle.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="NavigationObstacle" inherits="Node" version="3.4">
+	<brief_description>
+		3D Obstacle used in navigation for collision avoidance.
+	</brief_description>
+	<description>
+		3D Obstacle used in navigation for collision avoidance. The obstacle needs navigation data to work correctly. This can be done by having the obstacle as a child of a [Navigation] node, or using [method set_navigation]. [NavigationObstacle] is physics safe.
```

Comment:
```suggestion
		3D obstacle used in navigation for collision avoidance.
	</brief_description>
	<description>
		3D obstacle used in navigation for collision avoidance. The obstacle needs navigation data to work correctly. This can be done by having the obstacle as a child of a [Navigation] node, or using [method set_navigation]. [NavigationObstacle] is physics safe.
```